run--03b--postestimation-disaggregate-estimates.Rmd

```diff
@@ -219,6 +219,36 @@ apportion_ages <- function(low_month, high_month, age_desc, verbose = FALSE) {
        pop_count_se_upd  = keep_pct*pop_count_se,
      )
    
+   ### Prorate any overlap with ages 6-12
+   # /!\ Could arguably generalize this together with the above, since the
+   # single-age calculations above are special cases of this below
+   
+   high_month_6to12 <- 12*(12+1) # this is the  last month of age 12
+    low_month_6to12 <- 6*12 + 1  # this is the first month of age 6
```

Comment:
So.. the value set up by these two lines are ignored(?) anyway in line 231 where we use functions min and max... but I wonder if we should fix these lines as below:
high_month_6to12 <- 12*(12+1) - 1 # this is the last month of age 12
low_month_6to12 <- 6*12                # this is the first month of age 6